parsec/core/resources/default_pattern.ignore

```diff
@@ -0,0 +1,83 @@
+
+
+
+### Temporary extensions ###
+
+*.tmp
+*.temp
+*.swp
+
+
+### Linux ###
+
+# Temporary files
+*~
+
+# Temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# .nfs files are created when an open file is removed but is still being accessed
+.nfs*
+
+# Glib temporary files
+.goutputstream-*
+
+### macOS ###
+
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
+# Thumbnails
+._*
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.com.apple.timemachine.donotpresent
```

Comment:
thanks they didn't called this shit ` .com.apple.timemachine.dotnotpresent ` :trollface:

Reply:
:smile: